src/IssueViz/SelectionEvents/IAnalysisIssueSelectionEvents.cs

```diff
@@ -0,0 +1,66 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+
+namespace SonarLint.VisualStudio.IssueVisualization.SelectionEvents
+{
+    internal interface IAnalysisIssueSelectionEvents : IDisposable
+    {
+        event EventHandler<IssueChangedEventArgs> SelectedIssueChanged;
+        event EventHandler<FlowChangedEventArgs> SelectedFlowChanged;
+        event EventHandler<LocationChangedEventArgs> SelectedLocationChanged;
+
+        IAnalysisIssueVisualization SelectedIssue { get; set; }
```

Comment:
Does `Visualization`mean `ViewModel` in this context?

Reply:
Not really; `Visualization` are decorator objects, similar to `IFilterableIssue`. They will be the objects that get passed to the editor as well, and it seems odd to have the editor reference view models.

Reply:
The editor UI is tied to WPF so using the term view model doesn't seem odd to me.